src/main/java/racing/RacingGame.java

```diff
@@ -2,53 +2,74 @@
+import java.util.stream.IntStream;
 
 public class RacingGame {
 
     private final List<Car> cars;
-    private final MovingStrategy movingStrategy;
+    private final int tryNums;
 
-    public RacingGame(List<Car> cars, MovingStrategy movingStrategy) {
+    public RacingGame(List<Car> cars, int tryNums) {
         this.cars = cars;
-        this.movingStrategy = movingStrategy;
+        this.tryNums = tryNums;
     }
 
-    public RacingGame(String carNames) {
-        this(carNames.split(","));
+    public RacingGame(String carNames, int tryNums) {
+        this(carNames.split(","), tryNums);
     }
 
-    public RacingGame(String[] carNames) {
+    private RacingGame(String[] carNames, int tryNums) {
         this(Arrays.stream(carNames)
                 .map(Car::new)
-                .collect(Collectors.toList()));
-    }
-    public RacingGame(List<Car> cars) {
-        this(cars, new RandomStrategy());
+                .collect(Collectors.toList()), tryNums);
     }
 
-    public List<Car> getWinners() {
+    public List<String> getWinners() {
         Map<Integer, List<Car>> groupByPosition = cars.stream().collect(Collectors.groupingBy(Car::getPosition));
-        int winnerPosition = getWinnerPosition(groupByPosition);
-        return getWinnerSortedByName(groupByPosition, winnerPosition);
+        int winnerPosition = getMaxPosition(groupByPosition);
+        List<Car> winner =  getWinnerSortedByName(groupByPosition, winnerPosition);
+        return extreactNames(winner);
+    }
+
+    private List<String> extreactNames(List<Car> cars) {
+        return cars.stream()
+                .map(Car::getName)
+                .collect(Collectors.toList());
     }
 
-    private static int getWinnerPosition(Map<Integer, List<Car>> groupByPosition) {
+    private int getMaxPosition(Map<Integer, List<Car>> groupByPosition) {
         List<Integer> positions = new ArrayList<>(groupByPosition.keySet());
         positions.sort(Collections.reverseOrder());
         return positions.get(0);
     }
 
-    private static List<Car> getWinnerSortedByName(Map<Integer, List<Car>> groupByPosition, int winnerPosition) {
+    private List<Car> getWinnerSortedByName(Map<Integer, List<Car>> groupByPosition, int winnerPosition) {
         List<Car> winners = groupByPosition.get(winnerPosition);
         winners.sort(Comparator.comparing(Car::getName));
         return winners;
     }
 
-    public void moveCars() {
-        cars.forEach(car -> car.moveForward(movingStrategy.generateNumber()));
+    public List<List<CarInfo>> playRounds() {
+        List<List<CarInfo>> rounds = new ArrayList<>();
+        rounds.add(getCarsPositionInfos());
+        IntStream.range(0, tryNums).forEach(i -> {
+            moveCars();
+            rounds.add(getCarsPositionInfos());
+        });
+        return rounds;
+    }
+
+    private void moveCars() {
+        cars.forEach(car -> car.moveForward(new RandomStrategy().generateNumber()));
     }
```

Comment:
아마도 playRounds 메서드에대한 테스트가 없는 이유는 이 메서드가 테스트하기가 어려웠기 때문일 것 같아요. 🤔

아래와 같이 변경했다면 테스트하기힘든 random 요소를 가진 `moveStrategy`를  비즈니스 로직 밖으로 꺼낼 수 있고 
테스트할 수 있는 코드가 됐을 것 같아요. 😃  

```suggestion
    public List<List<CarInfo>> playRounds(MoveStrategy moveStrategy) {
        List<List<CarInfo>> rounds = new ArrayList<>();
        rounds.add(getCarsPositionInfos());
        IntStream.range(0, tryNums).forEach(i -> {
            moveCars(moveStrategy);
            rounds.add(getCarsPositionInfos());
        });
        return rounds;
    }

    private void moveCars(MoveStrategy moveStrategy) {
        cars.forEach(car -> car.moveForward(moveStrategy.generateNumber()));
    }
```